client/post-editor/post-editor.jsx

```diff
@@ -739,6 +778,12 @@ const PostEditor = React.createClass( {
 				this.getMessage( 'view' ),
 				this.state.post.URL
 			);
+		} else if ( 'future' === this.state.post.status && this.state.isAutosaving === false ) {
```

Comment:
The latter part of the condition could be simplified to `! this.state.isAutosaving`